app/requirements.txt

```diff
@@ -25,7 +25,7 @@ lazy-object-proxy==1.4.1
 MarkupSafe==1.1.1
 mccabe==0.6.1
 pluggy==0.12.0
-psycopg2==2.7.7
+psycopg2>=2.7,<2.8 --no-binary psycopg2
```

Comment:
I was under the impression that we wanted to specifically fix the versions to ensure our built containers are reproducible, so maybe we should make this `psycopg2==2.7.7 --no-binary psycopg2` instead (assuming that would work). Also, should we update to 2.8?